indico/modules/rb/__init__.py

```diff
@@ -40,7 +40,8 @@
     'vc_support_emails': [],
     'notification_hour': 6,
     'notification_before_days': 1,
-    'notifications_enabled': True
+    'notifications_enabled': True,
+    'booking_limit': 365
```

Comment:
Where's the field to change this setting? Also, I'd use a timedelta for it (check `TimedeltaConverter`)